### PR TITLE
[VEN-2548] Add Pendle weETH Market

### PR DESCRIPTION
**Pull request by web3rover**

### CODE REVIEW

File: deployed-contracts/isolated-pools.md [hunk 1]
```diff
@@ -291,6 +293,7 @@
 * Comptroller: [`0xd79CeB8EF8188E44b7Eb899094e8A3A4d7A1e236`](https://sepolia.etherscan.io/address/0xd79CeB8EF8188E44b7Eb899094e8A3A4d7A1e236)
 * NativeTokenGateway: [`0x1FD30e761C3296fE36D9067b1e398FD97B4C0407`](https://sepolia.etherscan.io/address/0x1FD30e761C3296fE36D9067b1e398FD97B4C0407)
 * Markets:
+  * * vPT-weETH-26DEC2024\_LiquidStakedETH: [`0x3AF2bE7AbEF0f840b196D99d79F4B803a5dB14a1`](https://sepolia.etherscan.io/address/0x3AF2bE7AbEF0f840b196D99d79F4B803a5dB14a1)
```
Comment (chechu) [inline deployed-contracts/isolated-pools.md]:
```suggestion
  * vPT-weETH-26DEC2024\_LiquidStakedETH: [`0x3AF2bE7AbEF0f840b196D99d79F4B803a5dB14a1`](https://sepolia.etherscan.io/address/0x3AF2bE7AbEF0f840b196D99d79F4B803a5dB14a1)
```

File: deployed-contracts/oracles.md [hunk 2]
```diff
@@ -40,6 +40,7 @@
 * weETH Oracle (assuming 1:1 for eETH:ETH):[`0xEa687c54321Db5b20CA544f38f08E429a4bfCBc8`](https://etherscan.io/address/0xEa687c54321Db5b20CA544f38f08E429a4bfCBc8)
 * weETH Oracle (not assuming 1:1 for eETH:ETH):[`0x660c6d8c5fddc4f47c749e0f7e03634513f23e0e`](https://etherscan.io/address/0x660c6d8c5fddc4f47c749e0f7e03634513f23e0e)
 * sFrax Oracle: [`0x27F811933cA276387554eAffD9860e513bA95AC3`](https://etherscan.io/address/0x27F811933cA276387554eAffD9860e513bA95AC3)
+* PendleOracle PTweETH-26DEC2024 Oracle: [`0x16D54113De89ACE580918D15653e9C0d1DE05C35`](https://etherscan.io/address/0x16D54113De89ACE580918D15653e9C0d1DE05C35)
```
Comment (chechu) [inline deployed-contracts/oracles.md]: `0xA1eA3cB0FeA73a6c53aB07CcC703Dc039D8EAFb4`?

File: risk/resilient-price-oracle.md [hunk 3]
```diff
@@ -141,6 +141,7 @@ The current list of correlated token oracles in Venus is:
 | Core | sFRAX | [SFraxOracle](https://etherscan.io/address/0x27F811933cA276387554eAffD9860e513bA95AC3) | - | - | |
 | Curve | crvUSD | [Chainlink](https://etherscan.io/address/0x94c3A2d6B7B2c051aDa041282aec5B0752F8A1F2) | - | - | |
 | Curve | CRV | [Chainlink](https://etherscan.io/address/0x94c3A2d6B7B2c051aDa041282aec5B0752F8A1F2) | - | - | |
+| Liquid Staked ETH | PTweETH-26DEC2024 | [PendleOracle-PT-weETH-26DEC2024](https://etherscan.io/address/0xAF83f9C9d849B6FF3A33da059Bf14A0E85493eb4) | - | - | |
```
Comment (chechu) [inline risk/resilient-price-oracle.md]: Wrong address?

Comment (web3rover) [reply risk/resilient-price-oracle.md]: Fixed

File: deployed-contracts/oracles.md (hunk 2 above)
Comment (web3rover) [reply deployed-contracts/oracles.md]: Fixed

**Pull request merged**